src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -262,3 +263,10 @@
      ;;  :cost () ;; tbd not used on desktop
      :token-fees                token-fees
      :gas-amount                (:tx-gas-amount new-path)}))
+
+(defn rpc->collectibles
+  [collectibles]
+  (->> collectibles
+       (cske/transform-keys transforms/->kebab-case-keyword)
+       (map #(assoc % :unique-id (collectible-utils/get-collectible-unique-id %)))
+       vec))
```

Comment:
In the future, having the collectibles as a hash map is gonna be easier than having a vector/list, since it's data that must be constantly updated